tests/integration/views/test_questionnaire.py

```diff
@@ -34,17 +36,16 @@ def tearDown(self):
         self._application_context.pop()
 
     def test_update_questionnaire_store_with_form_data(self):
-
         schema = load_schema_from_params('test', '0112')
-
         location = Location('rsi', 0, 'total-retail-turnover')
 
-        form_data = {
+        form_data = MagicMock(spec=QuestionnaireForm, data={
             'total-retail-turnover-answer': '1000',
-        }
+        })
 
         with self._application.test_request_context():
-            update_questionnaire_store_with_form_data(self.question_store, location, form_data, schema)
+            answer_store_updater = AnswerStoreUpdater(location, schema, self.question_store)
+            answer_store_updater.save_answers(form_data)
```

Comment:
Is this test relevant now that we have `AnswerStoreUpdater` and it's tests. This looks like it is duplicating tests.